test/integrations/sources/facebook_lead_ads_native/data.ts

```diff
@@ -15,15 +15,15 @@ export const data = [
                 object: 'page',
                 entry: [
                   {
-                    id: '112464600187556',
+                    id: '134587',
                     time: 1758794733,
                     changes: [
                       {
                         field: 'leadgen',
                         value: {
                           created_time: 1758794728,
                           leadgen_id: '1459076748710713',
-                          page_id: '112464600187556',
+                          page_id: '134587',
```

Comment:
Updating `page_id` value in the test has got nothing to do with the actual bug or the fix in this PR.